CAndC++.md

```diff
@@ -55,6 +55,10 @@ greater performance will be added soon after, including:
   [LLVM's vector extensions]: http://clang.llvm.org/docs/LanguageExtensions.html#vectors-and-extended-vectors
   [GCC's vector extensions]: https://gcc.gnu.org/onlinedocs/gcc/Vector-Extensions.html
 
+Note that non-WebAssembly inline assembly or platform-specific features
+are not supported, such as x86 assembly or MMX/SSE/NEON vectorization
+primitives.
```

Comment:
This is, ironically, still slightly misleading, since Emscripten provides <xmmintrin.h> and a few others that provide compatibility with x86-specific APIs, implemented via asm.js SIMD constructs (WebAssembly doesn't yet have SIMD, but it is expected to before long).